source/includes/facts-versioning.rst

```diff
@@ -0,0 +1,36 @@
+.. start-rewind-desc
```

Comment:
@poornas @vadmeste @kannappanr 

So I tried to refactor a bit, since these options are used quite widely. 

The following file is *referenced* by the various commands that use these options. Specifically, `.. start-<option>-desc` indicates the start of the section and
`.. end-<option>-desc` indicates the end of the section. Sphinx grabs only whats in the middle and then performs additional transformations later.

`|<string>|` gets replaced in the inheriting document. It should, based on my testing, always produce appropriate results.

The descriptions are a little more generic in this form, but its much easier to manage changes here.

If we do end up with specific commands that have more unique behavior, I can expand this file to include